SPEC.md

```diff
@@ -70,25 +70,25 @@ The key words "must", "must not", "required", "shall", "shall not", "should", "s
     "cdiVersion": "0.1.0",
     "kind": "<name>",
     "kindShort": ["<short-name>", ...],
-    "container-runtime": ["<container-runtime-name>"], (optional)
+    "containerRuntime": ["<container-runtime-name>"], (optional)
```

Comment:
why did we need this field for even though it is optional?

Reply:
This field is here so that a vendor can have a CDI specification for specific runtimes. e.g: the nvidia-kata.json and the nvidia.json specification